events/workshops-elmi-june-2021.html

```diff
@@ -0,0 +1,51 @@
+---
+layout: news-events
```

Comment:
Outside the scope of this PR and following the OME 2021 experience, we could investigate structuring the metadata for each upcoming workshop using schema.org and create a template allowing to render the event as HTML from the JSON.

Reply:
:heart: